.travis/post_docs_test.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env sh
+
+export BASE_ADDR=http://pulp:80
+export CONTENT_ADDR=http://pulp:80
+
+cd docs/_scripts/
+bash docs_check_upload.sh
+bash docs_check_sync.sh
```

Comment:
does it make sense to test the docs_check_install_with_signed_repodata.sh  as well?

Reply:
I tried but it was not possible because it tries to dnf install and travis uses ubuntu

Reply:
and I believe it makes sense to test it (and maybe other workflows), but I think it should be addressed on another issue, as it will require to run the scripts inside the container

Reply:
https://pulp.plan.io/issues/6783